setup.py

```diff
@@ -43,4 +43,13 @@
     python_requires=">=3.6,<4",
     url=f"https://github.com/ARMmbed/{PROJECT_SLUG}",
     version=__version__,
+    entry_points={
+        "console_scripts": [
+            f"mbedtoolsdevices={SOURCE_DIR}.mbed_tools.cli:cli",

```

Comment:
I'm wondering if trying to cover every possible hyphenated/non-hyphenated combination is overkill now. It sounded find when we were just talking about `mbedtools`, but this seems a bit over the top to me. Is there an increase in the package install time now we're creating all these entry points?

Reply:
good point. No idea.